community/browser/app/scripts/services/UDC.coffee

```diff
@@ -108,7 +109,7 @@ angular.module('neo4jApp.services')
             today = new Date()
             today = new Date(today.getFullYear(), today.getMonth(), today.getDay())
 
-            if (pingTime < today)
+            if (pingTime < today) || true
```

Comment:
What's the intention here?
This will always be entered, and that's why the build tests are failing.